src/python/bot/fuzzers/syzkaller/engine.py

```diff
@@ -113,9 +140,19 @@ def fuzz(self, target_path, options, unused_reproducers_dir=None, max_time=0):
     self._create_temp_corpus_dir('new')
 
     args = options.arguments
-    args += ['--coverfile', runner.get_cover_file_path()]
+    args += [
+        '--coverfile',
+        runner.get_cover_file_path(), '--input', options.corpus_dir
```

Comment:
i dont think we want to use
'--input', options.corpus_dir
since you are copying corpus to work dir and not using the one from corpus dir.

Reply:
Done.